### ISSUE TITLE
Update Installation Steps

### ISSUE THREAD

**Issue by qjhart** [UCDavisLibrary/fin-example-repository#34]
*The steps for installing the server have changed recently.  Here are some very quick notes that need to be added to current documentation.  Note the admin part is required for new systems too*

## Purging the system

The `damsctl` does not have a command to allow one to purge your dams.  In order to do that you need to use the docker-compose commands directly.  The steps below show this.

``` bash
# This sets up an alias to docker-compose, dams-dc
eval $(damsctl alias)
# You can erase your current setup, while maintaining your docker configuration with
dams-dc down -v
# Now you can bring this back up
dams-dc up -d 
```
At this point you have no admins for the site.   The biggest change is that now the admins are managed in the repositories ACL as well, so we have an new method of adding users.  This allows the rest of the configuration, even adding adding admins, to be done on any remote machine.

In order to add-admins without any users, you need to first login as the super-user.  This uses the JWT_SECRET in the server configuration file.

```bash
fin login --super-user qjhart@ucdavis.edu
# Here you add in the JWT_SECRET and JWT_ISSUER from your dams.conf file
```
Once logged in, you can add your admins with the `fin acl` command

```bash
for i in quinn jrmerz enebeker ladragoo; do
   fin acl add-admin ${i}@ucdavis.edu
done
```
Once you have these admins in place, you  use these to add new collections

``` bash
# Log in using your normal (in our case  CAS) ids.
fin login --headless
# create your new collection
cd ~/fin-example-repo/collection
fin io import ex1-pets .
```

**Comment by qjhart** [comment]
@ldragoon , These are the new steps for adding admins to the system.  The README needs to be updated.  Plesae pull a branch `new-admin` for this change.  I'm going to add it to the sprint as well.

**Comment by ldragoon** [comment]
Completed & Created a Pull request.

**Comment by qjhart** [comment]
@ldragoon, For commits follow these instructions, https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords.  Use an active tense as in *Update README, Fix #34* So these get closed automaically.
